src/Modules/LearningTasks/Tutor.LearningTasks.Core/UseCases/Authoring/LearningTaskService.cs

```diff
@@ -0,0 +1,67 @@
+﻿using AutoMapper;
+using FluentResults;
+using Tutor.BuildingBlocks.Core.UseCases;
+using Tutor.LearningTasks.API.Dtos.LearningTasks;
+using Tutor.LearningTasks.API.Public;
+using Tutor.LearningTasks.API.Public.Authoring;
+using Tutor.LearningTasks.Core.Domain.LearningTasks;
+using Tutor.LearningTasks.Core.Domain.RepositoryInterfaces;
+
+namespace Tutor.LearningTasks.Core.UseCases.Authoring;
+
+public class LearningTaskService : CrudService<LearningTaskDto, LearningTask>, ILearningTaskService
+{
+    private readonly ILearningTaskRepository _learningTaskRepository;
+    private readonly IAccessServices _accessServices;
+
+    public LearningTaskService(ILearningTaskRepository learningTaskRepository, IAccessServices accessServices,
+        ILearningTasksUnitOfWork unitOfWork, IMapper mapper) : base(learningTaskRepository, unitOfWork, mapper)
+    {
+        _learningTaskRepository = learningTaskRepository;
+        _accessServices = accessServices;
+    }
+
+    public new Result<LearningTaskDto> Get(int id)
+    {
+        LearningTask? learningTask = _learningTaskRepository.Get(id);
```

Comment:
I don't know. If there is some reason to enable just instructor who is course owner to see learning tasks I will add that check. I didn't add that check because read operation is not idempotent operation and can't cause any side effect.